src/Illuminate/Database/Query/Builder.php

```diff
@@ -753,6 +753,10 @@ public function prepareValueAndOperator($value, $operator, $useDefault = false)
             throw new InvalidArgumentException('Illegal operator and value combination.');
         }
 
+        if (strpos($operator, '?') !== false) {
+            $operator = str_replace('?', '??', $operator);
+        }
+
```

Comment:
Uh ... 
![image](https://user-images.githubusercontent.com/87493/80119359-e196ed80-8589-11ea-9a0e-6b942b951394.png)

Reply:
Why ? The RFC stipulates that anything containing a single question mark on the operator must be duplicated in order not to be considered as a binding. Are there cases where the operator cannot be a string?

Reply:
The assumption is that the user might not want to escape the question mark. But what happens if they have? Would the suggestion below be better? 

if ($operator === "?") {
    $operator = str_replace('?', '??', $operator);
}